docs/references/vod/upload.mdx

```diff
@@ -70,6 +71,56 @@ curl --location --request PUT "${url}" \
 --data-binary '@$VIDEO_FILE_PATH'
 ```
 
-Notice that we only support mp4 files encoded with H264 video and AAC audio for
-now. If you are unsure about these, try sending the file anyway and check any
+> **Notice** that we only support mp4 files encoded with H264 video and AAC audio at the moment. If you are unsure about these, try sending the file anyway and check any
 returned errors!
+
+## Step 2b: Resumable upload
+
+Livepeer Studio supports resumable uploads via [Tus](https://tus.io/).
+This section provides a simple example of how to use `tus-js-client` to upload a video file.
+
+From the previous section, we generated a URL to upload a video file to Livepeer on `POST /api/asset/request-upload`.
+We can use the `tusEndpoint` field of the response to upload the video file and track the progress:
+
+```jsx
+  //  This assumes there is an `input` element of `type="file"` with id `fileInput` in the HTML
+  const input = document.getElementById("fileInput");
+  const file = input.files[0];
+  const upload = new tus.Upload(file, {
+      endpoint: tusEndpoint, // URL from `tusEndpoint` field in the `/request-upload` response
+      metadata: {
+        filename,
+        filetype: "video/mp4",
+      },
+      uploadSize: file.size,
+      onError(err) {
+        console.error("Error uploading file:", err)
+      },
+      onProgress(bytesUploaded, bytesTotal) {
+        const percentage = ((bytesUploaded / bytesTotal) * 100).toFixed(2);
+        console.log("Uploaded " + percentage + "%");
+      },
+      onSuccess() {
+        console.log("Upload finished:", upload.url);
+      },
+    });
+  upload.start();
+```
+
+You can resume previously interrupted uploads, using the client:
+
+```jsx
+  upload.findPreviousUploads().then(function (previousUploads) {
+      // Found previous uploads so we select the first one. 
+      if (previousUploads.length) {
+          upload.resumeFromPreviousUpload(previousUploads[0])
+      }
+
+      // Start the upload
+      upload.start()
+  })
```

Comment:
I think we should just provide a single snippet that already handles resuming previous uploads, WDYT?
```suggestion
  const previousUploads = await upload.findPreviousUploads();
  if (previousUploads.length > 0) {
      upload.resumeFromPreviousUpload(previousUploads[0])
  }
  upload.start()
```